MANO-API/src/main/java/com/ubiqube/etsi/mano/service/UbiRest.java

```diff
@@ -9,10 +9,16 @@
 import org.springframework.stereotype.Service;
 import org.springframework.web.client.RestTemplate;
 import org.springframework.web.util.UriComponentsBuilder;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+
 
 @Service
+@PropertySource("classpath:ubi-mano.properties")
```

Comment:
We cannot have configuration inside a war, in this case we have to rebuild a war per env. That's something we would like to avoid.